classes/fields/datetime.php

```diff
@@ -784,55 +798,64 @@ public function convert_format( $source_format, $args = array() ) {
 		$args = array_merge(
 			array(
 				'source' => 'php',
+				'type'   => 'date',
 			// 'jquery_ui' for reverse.
 			), $args
 		);
 
 		// Keep keys and values sorted by string length.
-		$symbols = array(
-			// Day
-			'd' => 'dd',
-			'l' => 'DD',
-			'D' => 'D',
-			'j' => 'd',
-			'N' => '',
-			'S' => '',
-			'w' => '',
-			'z' => 'o',
-			// Week
-			'W' => '',
-			// Month
-			'F' => 'MM',
-			'm' => 'mm',
-			'M' => 'M',
-			'n' => 'm',
-			't' => '',
-			// Year
-			'L' => '',
-			'o' => '',
-			'Y' => 'yy',
-			'y' => 'y',
-			// AM/PM
-			'a' => 'tt',
-			'A' => 'TT',
-			// Swatch internet time (not supported)
-			'B' => '',
-			// Hour
-			'h' => 'hh',
-			'H' => 'HH',
-			'g' => 'h',
-			'G' => 'H',
-			// Minute
-			'i' => 'mm',
-			// Second
-			's' => 'ss',
-			// Microsecond
-			'u' => 'c',
-		);
+		if ( 'time' === $args['type'] || 'time' === static::$type ) {
+
+			$symbols = array(
+				// AM/PM
+				'a' => 'tt',
+				'A' => 'TT',
+				// Swatch internet time (not supported)
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 8174a7903460736eeeaa25be50701cb1</sub>